std_execution.bs

```diff
@@ -2825,24 +2826,24 @@ namespace std::execution {
 
   // [execution.schedulers.queries], scheduler queries
   enum class forward_progress_guarantee;
-  inline namespace <i>unspecified</i> {
+  namespace <i>unspecified</i> {
     struct forwarding_scheduler_query_t;
-    inline constexpr forwarding_scheduler_query_t forwarding_scheduler_query{};
 
     template &lt;class T>
       concept <i>forwarding-scheduler-query</i> = // exposition only
-        forwarding_scheduler_query(T{});
+        forwarding_scheduler_query_t{}(T{});
 
```

Comment:
Do we need to specify this concept? Is implementation detail.

Reply:
We can remove this exposition only concept.